tests/bud.bats

```diff
@@ -5143,6 +5143,21 @@ _EOF
   assert "$manifests" = "amd64 arm64 ppc64le s390x" "arch list in manifest"
 }
 
+# attempts to resolve heading arg as base-image with --all-platforms
+@test "bud-multiple-platform-with-base-as-default-arg" {
```

Comment:
Not needed. I have no time to go into it, but main is broken because of an accidental push, and the test was added there. Do a git log or ask Dan for details. The test is already there. Thanks for fixing this.

Reply:
@edsantiago I think this is a different test for a different feature which is not already being tested in upstream, see: https://github.com/containers/buildah/pull/4401#discussion_r1013865451